Specs/DateTime/Japanese/DatePeriodExtractor.json

```diff
@@ -742,25 +685,42 @@
   },
   {
     "Input": "3年後に不在にします。",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
-    "Results": []
+    "Results": [
```

Comment:
Please add a Comment tag in these specs to explain the ambiguity and the decision for the current behaviour.

Reply:
added